API.md

```diff
@@ -1,5 +1,9 @@
 # PDFTron React Native API
 
+## TypeScript
+
+PDFTron React Native now has support for TypeScript. Since not all customers use the language, the typings used in this document will be described using normal JavaScript types. For TypeScript users, type information is automatically provided while coding. They can also be found in our [TypeScript source files](src).
```

Comment:
not sure what's the significant of this paragraph here? what do you expect users to do after landing on a bunch of source code?

Reply:
clarified; it may be helpful if they want to see full list of the exact type aliases/interfaces